tests/test_integration.py

```diff
@@ -29,3 +33,100 @@ def test_backup_and_restore():
 def test_restore_no_backups():
     res = cli.main(["restore", "config", "--from=test"])
     assert res == "No valid backups found. Doing nothing."
+
+
+def test_schedule_and_cancel():
```

Comment:
I don't reall have unit tests for these new methods, but mocking in python is kinda painful and it feels a bit superfluous.